test/apparmor.bats

```diff
@@ -46,14 +46,15 @@ load_a_specific_apparmor_profile_as_default_apparmor_and_run_a_container_with_it
 	load_apparmor_profile "$APPARMOR_TEST_PROFILE_PATH"
 	start_crio "$APPARMOR_TEST_PROFILE_NAME"
 
-	sed -e 's/%VALUE%/apparmor-test-deny-write/g' "$TESTDATA"/sandbox_config_apparmor.json > "$TESTDIR"/apparmor2.json
-	sed -e 's/%VALUE%/apparmor-test-deny-write/g' "$TESTDATA"/container_redis_apparmor.json > "$TESTDIR"/apparmor_container2.json
+	jq '.linux.security_context.apparmor_profile = "apparmor-test-deny-write"' \ 
+	"$TESTDATA"/sandbox_config.json > "$TESTDIR"/apparmor2.json
```

Comment:
whitespace issue

Reply:
for some reason, make shfmt isn't letting me indent line 50, every time I do it just reverts back? Really weird.